### ISSUE TITLE
CVE-2021-23807

### ISSUE THREAD

**Issue by huineng** [GoogleChrome/workbox#2976]
workbox-webpack-plugin@6.3.0 requires jsonpointer@^4.1.0 via a transitive dependency on @apideck/better-ajv-errors@0.2.5

 "workbox-webpack-plugin": "^6.3.0",

```
# npm audit report

jsonpointer  <5.0.0
Severity: moderate
Prototype Pollution in node-jsonpointer - https://github.com/advisories/GHSA-282f-qqgm-c34q
fix available via `npm audit fix --force`
Will install workbox-webpack-plugin@6.1.5, which is a breaking change
node_modules/jsonpointer
  @apideck/better-ajv-errors  *
  Depends on vulnerable versions of jsonpointer
  node_modules/@apideck/better-ajv-errors
    workbox-build  >=6.2.0-alpha.0
    Depends on vulnerable versions of @apideck/better-ajv-errors
    node_modules/workbox-build
      workbox-webpack-plugin  >=6.2.0-alpha.0
      Depends on vulnerable versions of workbox-build
      node_modules/workbox-webpack-plugin
```

thanks

**Comment by jeffposnick** [comment]
This looks like it'll be addressed in https://github.com/apideck-libraries/better-ajv-errors/pull/6, at which point we'll publish v6.4.1 with an updated dependency on the new version of `@apideck/better-ajv-errors`.

**Comment by elsmr** [comment]
Hi @jeffposnick, just published `@apideck/better-ajv-errors@0.2.7`, it includes the dependency update of `jsonpointer` :smile:
